api/src/main/java/ca/bc/gov/educ/api/batchgraduation/processor/DistributionRunStatusUpdateProcessor.java

```diff
@@ -69,29 +69,31 @@ private Map<String, ServiceException> updateBackStudentRecords(List<StudentCrede
         Map<String, ServiceException> unprocessedStudents = new HashMap<>();
         String activityCode = getActivitCode(jobType);
         final int totalCount = cList.size();
-        final int[] processedCount = {0};
+        final int[] processedCount = {1};
         List<UUID> studentIDs = cList.stream().map(StudentCredentialDistribution::getStudentID).toList();
         cList.forEach(scd-> {
             try {
                 final String accessToken = restUtils.getAccessToken();
                 restUtils.updateStudentCredentialRecord(scd.getStudentID(),scd.getCredentialTypeCode(),scd.getPaperType(),
                         "NONGRADYERUN".equalsIgnoreCase(activityCode)? "IP" : scd.getDocumentStatusCode(),activityCode,accessToken);
-                LOGGER.debug("Dist Job [{}] / [{}] - update {} of {} student credential record: studentID, credentials, document status [{}, {}, {}]", batchId, activityCode, processedCount[0] + 1, totalCount, scd.getStudentID(), scd.getCredentialTypeCode(), scd.getDocumentStatusCode());
+                LOGGER.debug("Dist Job [{}] / [{}] - update {} of {} student credential record: studentID, credentials, document status [{}, {}, {}]", batchId, activityCode, processedCount[0], totalCount, scd.getStudentID(), scd.getCredentialTypeCode(), scd.getDocumentStatusCode());
                 processedCount[0]++;
             } catch (Exception e) {
                 unprocessedStudents.put(scd.getStudentID().toString(), new ServiceException(e));
+                LOGGER.error("Unexpected Error on update {} of {} student credential record: studentID [{}]", processedCount[0], totalCount, scd.getStudentID());
```

Comment:
Can you include the e.getMessage() in the logged error? Would be helpful to see if it was a result of a REST call issue from restUtils